posix-configs/SITL/init/ekf2/rover

```diff
@@ -23,20 +23,33 @@ param set SENS_BOARD_ROT 0
 param set SENS_DPRES_OFF 0.001
 param set SENS_BOARD_X_OFF 0.000001
 param set COM_RC_IN_MODE 1
-param set NAV_DLL_ACT 2
-param set NAV_ACC_RAD 2.0
-param set NAV_LOITER_RAD 10
+param set NAV_DLL_ACT 0
+param set NAV_ACC_RAD 0.5
+param set NAV_LOITER_RAD 2
 param set MIS_LTRMIN_ALT 0.01
 param set MIS_TAKEOFF_ALT 0.01
 param set FW_THR_IDLE 0.0
 param set EKF2_GBIAS_INIT 0.01
 param set EKF2_ANGERR_INIT 0.01
 param set EKF2_MAG_TYPE 1
 param set FW_AIRSPD_MIN 0
-param set FW_AIRSPD_TRIM 6
-param set FW_AIRSPD_MAX 8
-param set FW_PR_I 0.0
-param set FW_RR_I 0.0
+param set FW_AIRSPD_TRIM 4
+param set FW_AIRSPD_MAX 15.5
```

Comment:
"The TECS implementation provides a Total Energy Control System for fixed wing aircraft. Its outputs are throttle and pitch to achieve the desired sink / rise rates."

So TECS is for fixed wing. If the idea is to keep the code as an hack to FW controlling, this is ok. Otherwise, you need to find a way of providing throttle control to the system.